pushy/src/main/java/com/relayrides/pushy/apns/ApnsClientBuilder.java

```diff
@@ -349,6 +354,34 @@ public ApnsClientBuilder setGracefulShutdownTimeout(final long gracefulShutdownT
     }
 
     /**
+     * Sets the buffer allocator for the underlying netty channel
```

Comment:
Can we be a little more descriptive here and maybe explain why a user would want to do this, or what the consequences might be? Also, as a super minor thing, mind adding a period to the end of the sentence? ;)